Formula/opa.rb

```diff
@@ -0,0 +1,26 @@
+class Opa < Formula
+  desc "Open source, general-purpose policy engine"
+  homepage "https://www.openpolicyagent.org"
+  url "https://github.com/open-policy-agent/opa/archive/v0.10.1.tar.gz"
+  sha256 "6b6121cbf7efc42b6ebc3f5ba6a2533e4e6eaf4b9eec446fb6f8fce4e6c02ae7"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/open-policy-agent/opa").install buildpath.children
+
+    cd "src/github.com/open-policy-agent/opa" do
+      system "go", "build", "-o", bin/"opa", "-installsuffix", "static",
+                   "-ldflags",
+                   "-X github.com/open-policy-agent/opa/version.Version=#{version}"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    output = shell_output("#{bin}/opa 2>&1")
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
I added a small functional test.